go/porcelain/netlify_client.go

```diff
@@ -55,7 +55,9 @@ type Netlify struct {
 }
 
 func (n *Netlify) SetSyncFileLimit(limit int) {
-	n.syncFileLimit = limit
+	if limit > 0 {
+		n.syncFileLimit = limit
+	}
```

Comment:
This way, 0 value will take the default value given the way the buildbot initializes this client: https://github.com/netlify/buildbot/blob/5a312d8ce9ade738407c76651271f2b27353a6c9/api/api.go#L71-L72